metamorph/metamorph_api/metamorph_api.proto

```diff
@@ -85,10 +85,11 @@ message TransactionStatus {
   google.protobuf.Timestamp announced_at = 3;
   google.protobuf.Timestamp mined_at = 4;
   string txid = 5;
-  Status status = 6;
-  string reject_reason = 7;
-  uint64 block_height = 8;
-  string block_hash = 9;
+  string merkle_path = 6; 
```

Comment:
I think it would be better to add the merkle path at the end as the proto object becomes in compatible with a previous version.

Reply:
Do we need to be compatible? if not it's working rn